AU2/plugins/CorePlugin.py

```diff
@@ -237,7 +243,7 @@ def on_assassin_update(self, assassin: Assassin, htmlResponse: Dict) -> List[HTM
         return [Label("[CORE] Success!")]
 
     def on_event_request_create(self):
-        assassins = [(a.identifier, a.pseudonyms) for a in ASSASSINS_DATABASE.assassins.values()]
+        assassins = [(a.identifier, a.pseudonyms) for a in ASSASSINS_DATABASE.assassins.values() if not a.hidden]

```

Comment:
This should probably be extracted to a library method of `ASSASSINS_DATABASE` given it's also used in `on_event_request_update` - it might end up used elsewhere

Reply:
The whole thing? Or just the filtering out of hidden assassins?

Reply:
Fetching assassin ID/pseudonym pairs